spec/tags/language/symbol_tags.txt

```diff
@@ -0,0 +1,2 @@
+slow:A Symbol literal inherits the encoding of the magic comment and can have a binary encoding
+fails:A Symbol literal inherits the encoding of the magic comment and can have a binary encoding
```

Comment:
I looked into this a bit. For some reason, the lexer thinks the code range should be `CR_7BIT`. Debugging that part is going to take more effort. `SymbolParseNode` then goes on to encode the j.l.String to a rope with `USASCIIEncoding.INSTANCE`. That process converts that "bad" bytes to '?' and so the resulting rope is well-formed. It just no longer represents the same set of characters.

Reply:
Yes, the lexer definitely gets this wrong for some reason. And the encoding of the Symbol should be binary.